routes/Api/README.txt

```diff
@@ -0,0 +1 @@
+API routes can be placed here.
```

Comment:
I was not sure how far @rappasoft wanted to take this.   Was just doing the bare minimum while still making it easier.